### 问题类别分析
通过词云我们可以清楚地得到结论：`生活`、`心理学`、`恋爱`、`调查类问题`等问题出现的频率较高。但是具体数字不得而知，我们将进一步分析。

In [1]:
import pandas as pd

In [2]:
df = pd.read_csv(r'./info.csv')  # 导入csv文件

In [3]:
df['Type'].value_counts()

生活               204
调查类问题             76
心理学               39
恋爱                23
学习                23
人际交往              23
服饰搭配              20
美食                20
教育                19
美容护肤              18
健康                18
文学                18
人生                16
社会                16
创业                15
历史                13
生活方式              13
情感                12
设计                12
女性                11
健身                11
电影                10
旅游                 9
摄影                 8
X 是种怎样的体验          8
时尚                 8
生活常识               7
生活经历               6
互联网                6
电影推荐               6
                ... 
平面设计               1
亲密关系               1
精华回答               1
留学                 1
计算机网络              1
脑洞                 1
寝室神器               1
买房                 1
宠物                 1
考试                 1
化妆品                1
个人成长               1
英语翻译               1
硬笔书法               1
旅行分享               1
素质                 1
动漫           

In [4]:
top_10 = df['Type'].value_counts()  # 表格如下

| 类别  | 数量 |
| :---------: | :--------: |
| 生活  | 204 |
| 心理学  | 39 |
| 学习  | 23 |
| 恋爱  | 23 |
| 人际交往  | 23 |
| 美食  | 20 |
| 服饰搭配  | 20 |
| 教育  | 19 |
| 美容护肤  | 18 |
| 文学  | 18 |
| 健康  | 18 |

可以看出，`生活`类的问题占了1000个问题的五分之一，而2~9名的数量差距不大。光从数据来看，知乎成为了一个分享生活感悟、生活常识、人生哲理等生活类话题的社交平台，而不仅仅是一个专业知识问答社区。<br>
接下来，将要对每一个问题的`关注人数`以及`回答个数`进行分析。